Add a recipe for https://codeberg.org/treflip/zathura.el

tre-flip opened the pull request:
### Brief summary of what the package does
It's a set of simple commands that create hyperlinks (org-mode elisp links and Hyperbole buttons) to documents open in Zathura document viewer (zathura's dbus interface is used for this). Following such a link will open the linked document in a new instance of Zathura.

### Direct link to the package repository
https://codeberg.org/treflip/zathura.el

### Your association with the package

I'm the maintainer.

### Relevant communications with the upstream package maintainer

**None needed** 

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Heads up, you're using a combination of tabs and spaces for indentation, and I assume you have locally set your tabwidth to something tiny like 2.  Your code will look really strange to other people - *especially* if you're going to use a custom tab width, I'd recommend `(setq indent-tabs-mode nil)` in your personal config, at least for editing elisp code.

Your commentary:
```elisp
;;; Commentary:
;; A set of simple commands that create hyperlinks to documents open in Zathura document viewer.
```
should probably be rephrased as something like:
```elisp
;;; Commentary:

;; Zathura <https://pwmt.org/projects/zathura/> is a highly customizable and
;; functional document viewer, external to Emacs.  This package provides a set
;; of simple commands for creating hyperlinks to open documents in Zathura.
```

⸺ `zathura.el` with byte-compile using Emacs 30.1:
```
zathura.el:109:2: Warning: docstring wider than 80 characters
```

⸺ `zathura.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- `;;; Commentary` is much wider than 80 characters
```

- Add a GPL-compatible LICENSE file to the repository

tre-flip commented:
Thanks for your feedback! I don't remember changing any settings related to elisp formatting in my init.el. Anyway, I replaced tabs to spaces with `M-x untabify` and fixed commentary and docstring lengths. I've set `indent-tabs-mode` to nil to prevent wrong indentation in future.

See https://codeberg.org/treflip/zathura.el/commit/754664519f0c346a296d1fa81f3243c584eaef66
